README.md

```diff
@@ -252,6 +252,21 @@ When setting up a Load Balancer for HTTPS you will be required to set the `loadb
 
 The HTTPS Load Balancer also uses SSL termination at the Load Balancer level and will offload traffic to AWX over HTTP.
 
+  * NodePort
+
+The following variables are customizable only when `service_type=NodePort`
+
+| Name                           | Description                              | Default       |
+| ------------------------------ | ---------------------------------------- | ------------- |
+| nodeport_port            | Port used for NodePort       | 80            |
```

Comment:
Thanks for your contribution @BongoEADGC6. Could you please adjust the `80` to replace it with `30080` instead?

Reply:
Good eye. Updated.